### ISSUE TITLE
Address KU 

### ISSUE THREAD

**Issue by GuidoTurdera** [abjer/sds2019#14]
Hi! I'm arriving at CPH today - I read on the blog teaching takes place at 9AM in "lecture hall CSS 35-01-05". This is at Øster Farimagsgade 5, right? 

And as an external student should I do any in particular before class - any registration, maybe? 

Thanks!

Guido.

**Comment by Kristianuruplarsen** [comment]
> This is at Øster Farimagsgade 5, right?

Yes. Take a look at the [campus map](https://socialsciences.ku.dk/campusmap/).

> And as an external student should I do any in particular before class - any registration, maybe?

Don' t know. You should probably ask the administration.
